Jenkinsfile.deploy

```diff
@@ -1,5 +1,15 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
 pipeline {
-    agent { label 'operator-slave' }
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+
+    // Needed for local build
+    tools {
+        maven 'kie-maven-3.6.2'
```

Comment:
I think this can be removed as 3.6.3 is default for all kogito now.

Reply:
I don't like much implicit dependencies in fact.
For now it is 3.6.3 in the definition, but if the definition gets a new default version and that breaks everything, that will be a lot of maintenance to update all ...

Reply:
just thinking that it would be the same if we need to upgrade the maven version.
btw, why is not jdk 11 by default ?

Reply:
We can let the maven version stay in place, apparently, if you try `kie-maven-3.6.3` it doesn't work, needs to be configured on the tools plugin I guess, Though I don't think it is going to be downgraded as all the kogito is now working with 3.6.2+. Regarding Java 11, I'm not really sure It has been the default since the beginning with all java versions available, pipelines uses whichever version they require.

Reply:
Yeah, tool will now work with 3.6.3, so if you want we keep using 3.6.2 or upgrade it with 3.6.3

Reply:
+1 for remove it if the current maven version is 3.6.3

Reply:
ok will remove it

Reply:
removed